app/views/wiki/_wikis.html.erb

```diff
@@ -19,4 +20,5 @@
     </tr>
   <% end %>
 </table>
+
 <div class="text-center"><%= will_paginate @wikis, :renderer => BootstrapPagination::Rails if @paginated %></div>
```

Comment:
@gyi2521 looking awesome. Just two things we should change the `@wikis` here to `wikis`. 

Also refactor this file too https://github.com/publiclab/plots2/blob/master/app/views/questions/_questions.html.erb. Thanks

Reply:
@jywarren Thanks for putting titles together.  You are right!  I should have used separate branches but didn't do it for the second issue "Refactor wikis..".  So I end it up working with not the most recent version... @cesswairimu and that's why I didn't get to change the last part '<div class="text-center"><%= will_paginate @wikis' because the <div> didn't exist in the version I was working on, so I restored that code back after I noticed in master version.  Then, I ended up creating another branch and made PR again on: https://github.com/publiclab/plots2/pull/3918.
Hard learning for the first timer... ;-) 
I'll change the @wikis to wikis in the last <div> and will work on '_questions.html.erb' as well in a brand new branch and will make PR. ;-)
Thank you both!
-Gina

Reply:
@jywarren @cesswairimu 
Hi~  I am ready to push but I get the following message from my git bash:

nhyi@LAPTOP-QQ4VPR5V MINGW64 ~/bc/etc/plots2 (AddLoginPromptRefactorWikis)
$ git push origin AddLoginPromptRefactorWikis
remote: Permission to publiclab/plots2.git denied to gyi2521.
fatal: unable to access 'https://github.com/publiclab/plots2.git/': The requested URL re     turned error: 403

Any ideas?